sdk/core/azure-core/test/ut/CMakeLists.txt

```diff
@@ -42,8 +42,9 @@ add_executable (
     logging.cpp
     main.cpp
     nullable.cpp
+    operation.cpp
     pipeline.cpp
-    policy.cpp
+    policy.cpp    
```

Comment:
nit: revert extra spacing added here.

Reply:
LoL

![image](https://user-images.githubusercontent.com/24213737/102434317-5f9c7e80-3fc9-11eb-9f49-fb2e5d1d1bd8.png)

...

![image](https://user-images.githubusercontent.com/24213737/102434332-6925e680-3fc9-11eb-800f-8322deddf349.png)

Reply:
They keep coming back even though they aren't showing up in my VS changes,.  I believe they are gone now.

Reply:
It's still there :(